### ISSUE TITLE
As UX lead, I want the user to have a organized first experience for the Home page

### ISSUE THREAD

**Issue by abottoms-coder** [raft-tech/TANF-app#1496]
**Description:**
Per #653 , we will be updating our welcome page for a more concise

**Acceptance Criteria:** 
- [ ] “Welcome” page should be renamed “Home”
- [ ] This page displays the current "Request Access" form present in the old Profile page
- [ ] Once access is requested, will show message "Administrator will be in touch with you soon" per Figma link below
- [ ] Once user is approved, will show list of permissions like present
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
- [ ] Change "Welcome to TDP" to "Request Submitted" after submission
- [ ] Pull in "Request Access" form
- [ ] Ensure consistency with Figma Mockups
- [ ] Run Testing Checklist and confirm all tests pass

**Notes:**

**Supporting Documentation:**
- [Figma Mockups](https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=5892%3A48657)
- [![Video Demo](https://user-images.githubusercontent.com/28515874/146063861-57a9de59-8741-4517-8955-64138c470685.png)](https://user-images.githubusercontent.com/28515874/146063268-290ae9d2-9fd9-4db8-96cd-a7dae9169972.mp4 "Video Demo")
- 
**Open Questions:**

**Comment by stevenino** [comment]
Closed as a duplicate of #1495  per backlog refinement 4/19/22
